### DISCUSSION TITLE
小时候干过的蠢事 | /posts/silly-things/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#648] [Announcements]
https://yuanfan.rbind.io/posts/silly-things/

**Comment by shenweiyan** [comment]
勾起了很多童年的记忆，看的津津有味。

**Reply by earfanfan** [reply]
是咩，看的津津有味的话，这位朋友，你可不可以也给俺讲讲你的童年故事，我想着说不定能再勾起俺更多回忆。虽然我感觉我的童年回忆这几年已经被挖掘殆尽了。

**Comment by LFG12** [comment]
看到这篇我才知道是“凡凡姐姐”，前面我竟然以为是“凡凡哥哥”

**Reply by earfanfan** [reply]
这位朋友你好，我戳了一下你的头像，看到了你的个人简介，无意中发现你和我堂妹依依竟然是大学校友，哈哈，真是人生何处不相逢啊。

**Reply by LFG12** [reply]
哇，居然这么巧！这个回复在qq邮箱里压到了最下面刚刚看到，开学一定要见见这位学姐（或学妹）了哈哈

**Reply by earfanfan** [reply]
依依是你的学姐，因为她已经进入社会工作了。我跟她说好了，荷花开放的时候就再来看我，到时候喊你一起去玩哈。

**Reply by LFG12** [reply]
太棒了，期待春暖花开日的相见，YF姐

**Reply by earfanfan** [reply]
我跟依依都很好奇你们学校校区长撒样，正好今年端午假期不用调休，要不端午节三天假期里你挑一天带我们逛逛校园吧？
ps依依问，你们学校那么远，平时是不是很少出去玩呢。

**Reply by LFG12** [reply]
那看来依依学姐是在主校区，我12月去了一趟主校区，比武汉校区大多了，吃的也多，这里有一个720云的航拍可以云游校园可以看到校园风貌：https://www.720yun.com/t/b292cmpkcyv ；
只是目前还不好确定端午节有没有出去的安排（去年家乡聊城的半马在五一举办，今年有时间想参加一下，也有可能和朋友出去骑车去长沙或者荆州）当然还没有确定，如果端午节我还在学校的话十分欢迎你们过来（在武昌坐地铁+公交大概1.5h），还可以沿后官湖走走或者去附近的九真山登高

**Reply by LFG12** [reply]
回依依姐，确实是比较远，进城比较耗时间，不过我是喜欢往外跑的，大一到大三转了些地方，大四因为在考研，24年基本就没咋出去，
我还想到了一个视频：B站up主发的汉口两万步，里面展示了些老汉口的古建筑：https://www.bilibili.com/video/BV1w7CSYoEsY/ 在这漫步感觉也挺不错
还发现刚刚的720链接有个小问题：更正应该是这个https://www.720yun.com/t/b292cmpkcyv?scene_id=582651 ；否则原本的需要点击左下角全景列表手动切换到武汉校区

**Reply by ferryhahaha** [reply]
校友你好，我看了武汉校区的图，原来武汉校区真的四面环水，依山傍水的，操场那边临近很大一片湖，视野非常好，天气好的时候肯定很美。

**Reply by earfanfan** [reply]
哈哈，我来赛博介绍一下吧。

@LFG12  小友，你好，这位 ferryhahaha 就是拥有神奇脑回路的依依学姐。

@ferryhahaha  依依，这位 LFG12 就是最近与你提起的你的校友，是一位真诚热情的小朋友。

**Reply by ferryhahaha** [reply]
握爪

**Reply by earfanfan** [reply]
俺也伸个爪。@ferryhahaha 依依，你伸左爪还是右爪？

**Reply by ferryhahaha** [reply]
右爪

**Reply by earfanfan** [reply]
哦，那我也伸右爪。不过我脑子里刚才浮现一个画面，三个同时伸右爪进行握爪的人类，从空中俯视的话是一个电风扇，也或者说是写轮眼的形状。

ps 咱们握爪的同时走动起来，就变成了真的电风扇（吊扇）了，哈哈哈。

**Reply by LFG12** [reply]
俺也来了，一起握爪，真是奇妙的缘分，
武区这边确实风景不错，依山傍水，一个半岛插入后官湖，平常围着湖边绿岛漫步骑车都还挺舒服
当时也是冲着武汉两个字来的这儿，去了一次主校区后才知道学校的实力，让我重新选的话一定选荆州，小城生活太舒心了哈哈，物价又便宜又在市中心
另附一张老师拍的照片，第一眼都没认出来是我们学校
![武区风景](https://github.com/user-attachments/assets/1e15ab65-db5e-4af6-8090-4d1db3535821)

**Reply by LFG12** [reply]
哈哈哈，YF姐你描述的那个场面，我在B站视频里见过，搜共产握手第一个就是，再多人也能握的下
链接搬来放在下面：[共产握手](https://www.bilibili.com/video/BV15K411T7Ay/)

**Reply by earfanfan** [reply]
>YF姐你描述的那个场面，我在B站视频里见过

共产握手其实我是第一次见识到，昨天确实只是联想到了那种吊扇，就是吊在天花板上几个大叶片旋转吹风的那种。另外就是和火影忍者里面写轮眼的标识很像。又当然，你和依依都是零零后，大约没见过吊扇或者写轮眼。

**Reply by LFG12** [reply]
可惜自己没看过火影忍者，不知道写轮眼，不过大吊扇还是见过的，高中学校里夏天吹凉就靠吊扇了哈哈
